easyDiffractionApp/Gui/Pages/Home/MainContent.qml

```diff
@@ -171,6 +176,19 @@ Item {
         }
     }
 
+
+    function updateAppGuiGlobals(){
+        // Update globals for easyAppGui
+        EaGlobals.Variables.appIconPath = ExGlobals.Constants.appLogo
+        EaGlobals.Variables.essIconPath = ExGlobals.Constants.essLogo
+        EaGlobals.Variables.eulaUrl = ExGlobals.Constants.eulaUrl
+        EaGlobals.Variables.oslUrl = ExGlobals.Constants.oslUrl
+        EaGlobals.Variables.appUrl = ExGlobals.Constants.appUrl
+        EaGlobals.Variables.appPrefixName = ExGlobals.Constants.appPrefixName
+        EaGlobals.Variables.appSuffixName = ExGlobals.Constants.appSuffixName
+        EaGlobals.Variables.description = ExGlobals.Constants.description
+    }
+
```

Comment:
Instead of using global properties, one can create local ones (appUrl, etc.). In this case, all properties are defined only once when the dialog is created. See both `easyAppGui/Componenets/ProjectDescriptionDialog.qml` and `easyDiffractionApp/Gui/Pages/Project/ProjectDescriptionDialog.qml` as an example.